src/Common/Commands.Common/PowerBISettings.cs

```diff
@@ -10,45 +10,54 @@
 using System.Net.Http;
 using System.Reflection;
 using System.Runtime.Serialization.Json;
+using System.Text;
 using System.Threading.Tasks;
-using Microsoft.Extensions.Configuration;
+using System.Xml;
 using Microsoft.PowerBI.Common.Abstractions;
 using Microsoft.PowerBI.Common.Abstractions.Interfaces;
 
 namespace Microsoft.PowerBI.Commands.Common
 {
     public class PowerBISettings : IPowerBISettings
     {
-        private IConfigurationRoot Configuration { get; }
+        public const string FileName = "settings.json";
```

Comment:
I don't think you need to make this public anymore.

Reply:
I can make that private